app/src/theme/fonts.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * This file contains all application's style relative to fonts
+ */
+ import { StyleSheet } from 'react-native'
+ import { useTheme, Theme } from '@react-navigation/native';
+
+ enum FontSize {
+    'small' = 14,
+    'regular' = 16,
+    'large' = 24,
+}
+
+ export default function(theme : Theme): any {
```

Comment:
```suggestion
 export default function fonts(theme : Theme) {
```

případně nazvat to useFonts a pak můžeš použít useTheme hook uvnitř a nemusí ho komponenty dávat jako argument.

Return type by TS měl být schopen odvodit automaticky namísto `any`.

Reply:
💡